src/docfx/docset/Document.cs

```diff
@@ -67,6 +67,11 @@ internal class Document
         /// </summary>
         public string SiteUrl { get; }
 
+        /// <summary>
+        /// Gets the external accessible URL
+        /// </summary>
+        public string ExternalUrl { get; }
```

Comment:
This feels like `CanonicalUrlWithoutLocale`, see `BuildPage.GetCanonicalUrl`. Maybe also put `CanonicalUrl` here as well?

Reply:
sure

Reply:
Seems duplicated for me, what about just use CanonicalUrl for xref

---
In reply to: [245891513](https://github.com/dotnet/docfx/pull/3880#discussion_r245891513) [](ancestors = 245891513)